### DISCUSSION TITLE
Datadog Plugin - Add path as tag

### DISCUSSION THREAD

**Discussion by fritzrichter** [Kong/kong#7302] [Ideas and feature requests]
### Summary

It would be good to add the path as tag to the metrics in order to analyze and monitor individual API calls rather then aggregated metrics for the proxy.

### Example
We have a **service** called auth-service and a **route** which maps everything below /auth/ to that upstream integration. We retrieve metrics for that routing, but I would like to understand metrics per path such as **/auth/login** and **/auth/renew**. Instead of creating a lot of different metrics, we would rather like to use tags for that purpose like "path=/login".

**Comment by pdany1116** [comment]
Hi @fritzrichter, any news here? Have you found a way to work around this?
